tests/unit/plugins/modules/test_ec2_ami_info.py

```diff
@@ -0,0 +1,253 @@
+# (c) 2022 Red Hat Inc.
+
+# This file is part of Ansible
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from ansible.module_utils.common.dict_transformations import camel_dict_to_snake_dict
+from unittest.mock import MagicMock, patch, ANY, call
+import botocore.exceptions
+import pytest
+
+from ansible_collections.amazon.aws.plugins.modules import ec2_ami_info
+
+module_name = "ansible_collections.amazon.aws.plugins.modules.ec2_ami_info"
+
+
+@pytest.mark.parametrize("executable_users,filters,image_ids,owners,expected", [
```

Comment:
`executable_users` has only one value, this can be removed from the parameterized or add other values to increase coverage

Reply:
Added a assertion with `executable_users`.